Assets/MRTK/Providers/LeapMotion/LeapMotionDeviceManagerProfile.cs

```diff
@@ -44,6 +44,90 @@ public Vector3 LeapControllerOffset
+            " The LeapVRDeviceOffsetMode property is only taken into account if the LeapControllerOrientation is Headset.")]
+        private LeapVRDeviceOffsetMode leapVRDeviceOffsetMode = LeapVRDeviceOffsetMode.Default;
+
+        /// <summary>
+        /// The VR offset mode determines the calculation method for Leap Motion Controller placement while in VR. 
+        /// LeapVRDeviceOffsetModes:
+        ///     Default - No offset is applied to the controller.
+        ///     Manual Head Offset - Three new properties with a range constraint control the offset, LeapVRDeviceOffsetY, LeapVRDeviceOffsetZ and LeapVRDeviceOffsetTiltX.
+        ///     Transform - The new Leap Controller origin is set to a different transform.
+        /// The LeapVRDeviceOffsetMode property is only taken into account if the LeapControllerOrientation is Headset.
+        /// </summary>
+        public LeapVRDeviceOffsetMode LeapVRDeviceOffsetMode
+        {
+            get => leapVRDeviceOffsetMode;
+            set => leapVRDeviceOffsetMode = value;
+        }
+
+        [Range(-0.5f, 0.5f)]
+        [SerializeField]
+        [Tooltip("The Y-axis offset of the Leap Motion controller if the LeapVRDeviceOffsetMode is Manual Head Offset and the LeapControllerOrientation is Headset.  This property and the range " +
+            "constraints mirror the range specified in the LeapXRServiceProvider. ")]
+        private float leapVRDeviceOffsetY = 0.0f;
+
+        /// <summary>
+        /// The Y-axis offset of the Leap Motion controller if the LeapVRDeviceOffsetMode is Manual Head Offset and the LeapControllerOrientation is Headset.  This property and the range 
+        /// constraints mirror the range specified in the LeapXRServiceProvider. 
+        /// </summary>
+        public float LeapVRDeviceOffsetY
+        {
+            get => leapVRDeviceOffsetY;
+            set => leapVRDeviceOffsetY = value;
+        }
+
+        [Range(-0.5f, 0.5f)]
+        [SerializeField]
+        [Tooltip("The Z-axis offset of the Leap Motion controller if the LeapVRDeviceOffsetMode is Manual Head Offset and the LeapControllerOrientation is Headset.  This property and the range " +
+            "constraints mirror the range specified in the LeapXRServiceProvider. ")]
+        private float leapVRDeviceOffsetZ = 0.0f;
+
+        /// <summary>
+        /// The Z-axis offset of the Leap Motion controller if the LeapVRDeviceOffsetMode is Manual Head Offset and the LeapControllerOrientation is Headset.  This property and the range 
+        /// constraints mirror the range specified in the LeapXRServiceProvider. 
+        /// </summary>
+        public float LeapVRDeviceOffsetZ
+        {
+            get => leapVRDeviceOffsetZ;
+            set => leapVRDeviceOffsetZ = value;
+        }
+
+        [Range(-90, 90)]
+        [SerializeField]
+        [Tooltip("The X-axis tilt offset of the Leap Motion Controller if the LeapVRDeviceOffsetMode is Manual Head Offset and the " +
```

Comment:
Question for myself - degrees vs radians. Degrees I assume are more common, which is why we don't specify units here right? I think for distance things are generally understood to be meters, maybe for angles it's the same

Reply:
Yes the assumption is that the X -axis tilt is in degrees, I followed the constraints that were in the LeapXRServiceProvider.